app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/viewmodel/DrawerViewModel.java

```diff
@@ -91,8 +91,12 @@ private boolean isNewLayer(VectorLayerPresentation vlp) {
         }
 
         private void addLayerMenuItem(VectorLayerPresentation vlp) {
-            int menuId = mStringToIntegerMapper.put(vlp.getId());
-            mView.addToMenu(vlp.getName(), menuId);
+            int menuItemId = mStringToIntegerMapper.put(vlp.getId());
```

Comment:
Reading this line made me wonder alot.
The field's name is really confusing, it should be named in a way that says more than just string to int.
It's used to generate menu ids from strings ids, so maybe it should be called IdGenerator and the put message can be renamed to generate.